docs/source/recipes/express.md

```diff
@@ -42,6 +42,41 @@ if (__SERVER__) {
 }
 ```
 
+## Static Middleware
+
+The `staticMiddleware` is for serving static files (such as stylesheets and client-side JavaScript files) in the `BUILD_DIR/PUBLIC` or `PUBLIC_DIR` directory. It uses the `express.static()` function to serve static files and the `setHeaders()` function to add response headers to the files that it serves.
+
+### Function `setHeaders(path)`
+
+The `setHeaders()` function is used to update the response headers for a file. It takes the path of the file being served as an argument and adds the response headers to that file. It uses the `config` object specified in `config.settings.serverConfig.staticFiles` to determine which response headers should be added to the file.
+
+### Configuration
+
+The `config.settings.staticFiles` is an array of objects with three properties:
+
+- `id`: a string identifier for the static file rule
+- `match`: a regular expression that evaluates the path of the requested resource
+- `headers`: an object containing the headers added if the match is successful.
+
+To add response header Cache-Control: public, max-age=3600 to a file named styles.css located in BUILD_DIR/PUBLIC directory:
+
+```js
+
+settings.staticFiles = [
+    ...,{
```

Comment:
This code example should be run through whatever JavaScript linter Volto uses. I think this example would be reformatted.